docs/how-to/get-package-source.rst

```diff
@@ -1,194 +1,241 @@
-.. _get-package-source:
-
-===========================
-Get the Source of a Package
+Get the source of a package
 ===========================
 
-.. caution::
-
-    Work in progress!
-
-Before you can work on a packge you need to get the source code of a given 
-package. This article presents multiple ways how you can achive this.
+Before you can work on a :term:`source package<Source Package>` you need to get
+the :term:`source code<Source Code>` of that package. This article presents
+four ways to achieve this: :command:`git-ubuntu`, :command:`pull-pkg`, and
+:command:`apt-get source`, and :command:`dget`.
 
-``git-ubuntu``
---------------
+git-ubuntu
+----------
 
 .. note::
 
-   ``git-ubuntu`` is the modern way of working with ubuntu source packages.
+   ``git-ubuntu`` is the modern way of working with :term:`Ubuntu`
+   source packages.
 
 .. warning::
 
-    ``git-ubuntu`` is still in active development. It is possible that you 
-    run into rough edges or unsupported edge cases. You can ask for help
-    in the #ubuntu-devel channel or open a bug report on launchpad.
+    ``git-ubuntu`` is still in active development and these instructions
+    will likely change over time. While ``git-ubuntu`` will become the
+    default packaging method, for now you may encounter rough edges or
+    unsupported edge cases. You can ask for help in the ``#ubuntu-devel``
+    channel or `open a bug report <GitUbuntuBugs_>`_ on :term:`Launchpad`.
+    Bug reports are very welcome!
+
+Install
+~~~~~~~
+
+The following command will install ``git-ubuntu``:
+
+.. code-block:: bash
+
+    sudo snap install --classic --edge git-ubuntu 
 
-Install::
+Basic usage
+~~~~~~~~~~~
 
-    $ sudo snap install --classic --edge git-ubuntu 
+To clone a source package git repository to a directory:
 
-Basic usage::
+.. code-block:: bash
 
-    # Clone a source package git repository to a directory
-    $ git ubuntu clone PACKAGE [DIRECTORY]
+    git ubuntu clone PACKAGE [DIRECTORY]
 
-    # Generate the orig tarballs for a given source package
-    $ git-ubuntu export-orig
 
-Example::
+To generate the orig tarballs for a given source package:
```

Comment:
Should we have a glossary entry for "orig tarballs"?

Reply:
There actually is already an glossary entry ^^

I added the term ref; resolved with 3e0791b1877525cec82c0e675a5df2d955bd5d1a